### PR TITLE
feat: Add setAppID

### PR DESCRIPTION
**Pull request by torbenkikkert**
Adding wrapper for setAppID to RN library

### CODE REVIEW

**Comment by mikehardy** [comment]
Oh, I didn't see any change to the README that would help people discover this feature? A README sentence somewhere in the initialization section would help I think? https://github.com/thebergamo/react-native-fbsdk-next/blob/master/README.md#sdk-initialization

File: src/FBSettings.js [hunk 1]
```diff
@@ -58,4 +58,10 @@ module.exports = {
   initializeSDK() {
     Settings.initializeSDK();
   },
+  /**
+   * Set app id
+   */
+   setAppID(appID: string) {
+    Settings.setAppID(appID);
```
[the remark this answers is not in the document]
Comment (torbenkikkert) [reply src/FBSettings.js:69]: Good point. Will validate argument.

**Comment by torbenkikkert** [comment]
> Oh, I didn't see any change to the README that would help people discover this feature? A README sentence somewhere in the initialization section would help I think? https://github.com/thebergamo/react-native-fbsdk-next/blob/master/README.md#sdk-initialization

Have added section on setAppID to the readme. If there is anything else that need changed or added just let me know.

File: src/FBSettings.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (mikehardy) [reply src/FBSettings.js:69]: looks great! thank you

**Review by mikehardy — APPROVED** [review]
This looks great to me, good doc, validation is solid I think; I believe a lot of people will like this feature even for single apps and for people with multiple apps of course it will be great. Thanks!

**Comment by mikehardy** [comment]
I'd like to make sure @thebergamo doesn't see anything I'm missing but I tagged it as pending merge because in the absence of any other feedback in a couple days, I'll merge. That leads me to say using patch-package between now and a merge+release should be safe, I'm certain this exact code or perhaps some tiny modification will make it to release so if you do a local patch it'll converge shortly

:trophy:

**Comment by thebergamo** [comment]
It seems solid for me :)

Yeah, that's a very nice thing to have as sometimes some of us could get a bit confused with this native part.

It seems good to me to be merged.

Great work and thanks @torbenkikkert o/

**Comment by torbenkikkert** [comment]
Thanks for the good feedback and help getting this little feature in.

**Pull request merged**